src/main/java/com/andersenlab/lesson2/TicketService.java

```diff
@@ -19,5 +20,20 @@ public static void main(String[] args) {
         System.out.println(empty);
         System.out.println(full);
         System.out.println(limited);
+
+        storage.add(empty);
+        storage.add(full);
+        storage.add(limited);
+
+        System.out.println(getTicketBySector(StadiumSector.B));
+    }
+
+    private static Ticket getTicketBySector (StadiumSector sector){
+        for (Ticket ticket : storage) {
+            if (ticket.getStadiumSector() == sector){
+                return ticket;
+            }
+        }
+        return null;
```

Comment:
Return null from method like we have here it is not a best practice. It is recommended to use alternatives like Optional (in Java 8 and above) or throwing exceptions where appropriate.